src/Output/JsonOutput.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tomaj\NetteApi\Output;
+
+use JsonSchema\Validator;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Tomaj\NetteApi\Response\ResponseInterface;
+
+class JsonOutput extends Output
+{
+    private $schemaValidator;
+
+    private $code;
+
+    private $schema;
+
+    public function __construct(int $code, $schema)
+    {
+        $this->schemaValidator = new Validator();
+        $this->code = $code;
+        $this->schema = $schema;
+    }
+
+    public function validate(ResponseInterface $response): bool
+    {
+        if (!$response instanceof JsonApiResponse) {
+            return false;
+        }
+        if ($this->code !== $response->getCode()) {
```

Comment:
tu mozme len 1 kod povazovat za spravny?

Reply:
kazdy endpoint moze mat nadefinovane pole outputov ktore moze dosiahnut a realny vystup sa kontroluje voci vsetkym nadefinovanym, ak nesedi kod, tak nebudeme riesit schemu a ukoncime to rovno

Reply:
no ak by bol nejaky vystup validny voci viacerym kodom? mozno len daco nechapem :-(

Reply:
outputs je array, cize ak mozes mat ten isty vystup s viacerymi kodmi (co si uplne neviem predstavit), tak ho das do pola viackrat:

```
public function outputs(): array
{
    $schema = '{"type": "object"}';
    return [
        new JsonOutput(200, $schema),
        new JsonOutput(200, $otherSchema),
        new JsonOutput(400, $schema),
        new JsonOutput(404, $schema)
    ];
}
```

Ak by bol response code 200, tak sa kontroluju prve dva ci sedi schema voci datam v response (ostatne dva sa na tomto kroku ukoncia). Ak ani jeden nesedi, hodime error, ak aspon jeden sedi, tak mame validny output.

Keby bol response code 403, vsetky 3 nase nadefinovane outputy skoncia na tomto riadku lebo nesedi ocakavany kod.

Reply:
uz rozumiem ;-) cool